.github/main.workflow

```diff
@@ -0,0 +1,27 @@
+workflow "New workflow" {
+  on = "push"
+  resolves = ["GitHub Action for Docker"]
+}
+
+action "Master Branch Filter" {
+  uses = "actions/bin/filter@ec328c7554cbb19d9277fc671cf01ec7c661cd9a"
```

Comment:
did you create this file manually? Or use some UI for this?

where did you get these IDs (i.e. `ec328c7554cbb19d9277fc671cf01ec7c661cd9a`) from?

Reply:
Mostly UI, though I have done a bit manually:
![screen shot 2019-02-12 at 12 50 44 pm](https://user-images.githubusercontent.com/327948/52660244-e9a6c800-2ec4-11e9-8416-ee0d6b37558b.png)

I'll give you a demo at some point today